extension/src/plots/webview/messages.ts

```diff
@@ -244,19 +275,34 @@ export class WebviewMessages {
 
     const buildMetricOrParamPath = (type: string, path: string) =>
       type + FILE_SEPARATOR + path
-    const newOrder = reorderObjectList(plotIds, customPlots, 'id').map(
-      ({ metric, param }) => ({
-        metric: buildMetricOrParamPath(ColumnType.METRICS, metric),
-        param: buildMetricOrParamPath(ColumnType.PARAMS, param)
-      })
+    const newOrder: CustomPlotsOrderValue[] = reorderObjectList(
+      plotIds,
+      customPlots,
+      'id'
+    ).map(plot =>
+      isCheckpointValue(plot)
```

Comment:
The check seems useless here. `metric` is the same, `type` already exists as you're checking for it here, `param` can be set to undefined depending on the `type`

Reply:
This part is simplified now by adding a param to `Checkpoint` type plot values. I did look into making the param optional but it seemed to cause extra code complexity elsewhere.